Dockerfile

```
@@ -0,0 +1,70 @@
+# -------------------------- Dev ---------------------------------------
+
+FROM node:22-bullseye AS dev
+
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends git \
+    && rm -rf /var/lib/apt/lists/* \
+    && npm install -g yarn@1.22.19 --force \
+    && git config --global --add safe.directory /code
+
+WORKDIR /code
+
+# Build stage for web app
+FROM dev AS web-app-serve-build
+
+COPY ./package.json ./yarn.lock /code/
+
+RUN yarn install
+COPY . /code/
+
+ENV VITE_FIREBASE_API_KEY=FIrebaseAP1k3y
+ENV VITE_FIREBASE_AUTH_DOMAIN=FIrebaseAP1k3
+ENV VITE_FIREBASE_DATABASE_URL=https://mapswipe.org
+ENV VITE_FIREBASE_PROJECT_ID=FIrebaseAP13
+ENV VITE_FIREBASE_STORAGE_BUCKET=FIrebase1k3y
+ENV VITE_FIREBASE_MESSAGING_SENDER_ID=FIreAP1k3y
+ENV VITE_FIREBASE_APP_ID=FIrebaseAPy
+ENV VITE_COMMUNITY_DASHBOARD_URL=https://mapswipe.org
```

These examples should reflect the format of actual values

eg:

```suggestion
ENV VITE_FIREBASE_API_KEY=ExampleF1rebaseAP1k3y
ENV VITE_FIREBASE_AUTH_DOMAIN=example-key.firebaseapp.com
ENV VITE_FIREBASE_DATABASE_URL=https://example-database.firebaseio.com
ENV VITE_FIREBASE_PROJECT_ID=example
ENV VITE_FIREBASE_STORAGE_BUCKET=example.appspot.com
ENV VITE_FIREBASE_MESSAGING_SENDER_ID=123123456123
ENV VITE_FIREBASE_APP_ID=1:23456789:web:1abc234def567
ENV VITE_COMMUNITY_DASHBOARD_URL=https://mapswipe.org
```